doc/sphinx/package_service_mapping.json

```diff
@@ -868,7 +896,121 @@
   "azure-mgmt-appconfiguration": {
     "category": "Management",
     "service_name": "App Configuration",
-    "manually_generated": true
+    "namespaces": [
+      "azure.mgmt.appconfiguration"
+    ]
+  },
+  "azure-mgmt-privatedns": {
+    "category": "Management",
+    "service_name": "Private DNS",
+    "namespaces": [
+      "azure.mgmt.privatedns"
+    ]
+  },
+  "azure-mgmt-frontdoor": {
+    "category": "Management",
+    "service_name": "Network",
+    "namespaces": [
+      "azure.mgmt.frontdoor"
+    ]
+  },
+  "azure-mgmt-hybridcompute": {
+    "category": "Management",
+    "service_name": "Hybrid Compute",
+    "namespaces": [
+      "azure.mgmt.hybridcompute"
+    ]
+  },
+  "azure-mgmt-healthcareapis": {
+    "category": "Management",
+    "service_name": "Healthcare APIs",
+    "namespaces": [
+      "azure.mgmt.healthcareapis"
+    ]
+  },
+  "azure-mgmt-edgegateway": {
+    "category": "Management",
+    "service_name": "Edge Gateway",
+    "namespaces": [
+      "azure.mgmt.edgegateway"
+    ]
+  },
+  "azure-mgmt-deploymentmanager": {
+    "category": "Management",
+    "service_name": "Deployment Manager",
+    "namespaces": [
+      "azure.mgmt.deploymentmanager"
+    ]
+  },
+  "azure-mgmt-peering": {
+    "category": "Management",
+    "service_name": "Peering",
+    "namespaces": [
+      "azure.mgmt.peering"
+    ]
+  },
+  "azure-mgmt-appplatform": {
+    "category": "Management",
+    "service_name": "App Service",
+    "namespaces": [
+      "azure.mgmt.appplatform"
+    ]
+  },
+  "azure-mgmt-botservice": {
+    "category": "Management",
+    "service_name": "Bot Service",
+    "namespaces": [
+      "azure.mgmt.botservice"
+    ]
+  },
+  "azure-mgmt-costmanagement": {
+    "category": "Management",
+    "service_name": "Cost Management",
+    "namespaces": [
+      "azure.mgmt.costmanagement"
+    ]
+  },
+  "azure-mgmt-attestation": {
+    "category": "Management",
+    "service_name": "Attestation",
+    "namespaces": [
+      "azure.mgmt.attestation"
+    ]
+  },
+  "azure-mgmt-datashare": {
+    "category": "Management",
+    "service_name": "DataShare",
+    "namespaces": [
+      "azure.mgmt.datashare"
+    ]
+  },
+  "azure-mgmt-serialconsole": {
+    "category": "Management",
+    "service_name": "Serial Console",
+    "namespaces": [
+      "azure.mgmt.serialconsole"
+    ]
+  },
+  "azure-mgmt-storagesync": {
+    "category": "Management",
+    "service_name": "Storage",
+    "namespaces": [
+      "azure.mgmt.storagesync"
+    ]
+  },
+  "azure-mgmt-storagecache": {
+    "category": "Management",
+    "service_name": "Storage",
+    "namespaces": [
+      "azure.mgmt.storagecache"
+    ]
+  },
+  "azure-mgmt-vmwarecloudsimple": {
+    "category": "Management",
+    "service_name": "Virtual Machines",
```

Comment:
Not sure what the service name should be - I found the reference for CloudSimple here = https://azure.microsoft.com/en-us/services/azure-vmware-cloudsimple/
But not here - https://azure.microsoft.com/en-us/services/
I wrote this in JS - `VMware CloudSimple` But not sure what the right service name should be. Let me know whatever we should keep, and if need be, I'll change that for JS too

Reply:
Gonna put this under `VMWare by CloudSimple`

Which goes along with https://azure.microsoft.com/en-us/overview/azure-vmware/